src/main/resources/application-dev.yaml

```
@@ -0,0 +1,26 @@
+server:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)